spec/models/organization_spec.rb

```diff
@@ -11,8 +11,8 @@
   end
 
   it 'sends slack notification on organization creation' do
-    notifier = instance_double(Slack::Notifier)
-    allow(Slack::Notifier).to receive(:new).and_return(notifier)
+    notifier = class_double(SlackNotifier).as_stubbed_const(transfer_nested_constants: true)
+    allow(SlackNotifier).to receive(:ping)
```

Comment:
`expect(SlackNotifier).to receive(:ping).with(<what's message is expected>).once`